File: Back/BackEnd/BackEnd/DTO/UsuarioPosicaoDTO.cs
using BackEnd.Interfaces;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BackEnd.DTO
{
    public class UsuarioPosicaoDTO : IUsuarioPosicaoDTO
    {
        public Int64 ID { get; set; }
        public string Usuario_Nome { get; set; }
        public decimal Posicao_Latitude { get; set; }
        public decimal Posicao_Longitude { get; set; }
    }
}

File: Front/localizaFront/src/app/DTO/usuario-posicao-dto.ts
import { IUsuarioPosicaoDTO } from '../Interface/iusuario-posicao-dto';

export class UsuarioPosicaoDTO implements IUsuarioPosicaoDTO {
    id: number;
    usuario_Nome: string;
    posicao_Latitude: number;
    posicao_Longitude: number;
}
